scripts/stop_dynamodb.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# DynamoDB Local Stop Script
+echo "🛑 Stopping DynamoDB Local..."
+
+# Get the script directory
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Stop DynamoDB Local using Docker Compose
+cd "$SCRIPT_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)